### PR TITLE
Fix 'Unexpected Wazuh version' error

### PR DESCRIPTION
**Pull request by AlexRuiz7**
Solves an issue where the App read the wrong metadata as the absolute path to the App .conf files was to specified to the Splunk CLI

### CODE REVIEW

File: SplunkAppForWazuh/bin/utils/config.py [hunk 1]
```diff
@@ -16,11 +16,56 @@
...
+    --------
+        A dictionary with the configuration block requested from the file.
+    """
+    try:
+        app_conf = cli.getAppConf(
+            confName="app", 
+            app="SplunkAppForWazuh", 
+            use_btool=False, 
+            app_path=WAZUH_HOME
+        )
```
Comment (AlexRuiz7) [inline SplunkAppForWazuh/bin/utils/config.py:60]: Unused. I was testing and forgot to remove this.

**Review by asteriscos — APPROVED** [review]
Test: :heavy_check_mark: 
CR: :heavy_check_mark: 

Tested it installing other Splunk apps and checking the Wazuh version mismatch.

before the PR
image

After
image

File: SplunkAppForWazuh/bin/utils/config.py (hunk 1 above)
Comment (AlexRuiz7) [reply SplunkAppForWazuh/bin/utils/config.py:60]: The `getAppConf()` function returns the following data:

```python
{
    "default": {},
    "ui": {
        "is_visible": "1",
        "label": "Wazuh"
    },
    "launcher": {
        "version": "4.3.4",
        "author": "info@wazuh.com",
        "description": "Wazuh helps you to gain deeper security visibility into your infrastructure by monitoring hosts at an operating system and application level."
    },
    "package": {
        "id": "SplunkAppForWazuh",
        "check_for_updates": "1"
    },
    "install": {
        "build": "4307"
    },
    "triggers": {
        "reload.package": "simple",
        "reload.config": "simple"
    }
}
```

**Review by Desvelao — APPROVED** [review]
**review**
code :heavy_check_mark: 
test :heavy_check_mark: 

## Screenshots
image
image
> Splunk with some apps

**Pull request merged**